vault/keyring.go

```diff
@@ -205,9 +240,15 @@ func (k *Keyring) Zeroize(keysToo bool) {
 	}
 }
 
-func (k *Keyring) AddEncryptionEstimate(term uint32, delta uint64) {
-	key := k.TermKey(term)
-	if key != nil {
-		atomic.AddUint64(&key.Encryptions, delta)
```

Comment:
Since we don't use atomic here anymore, the import sync/atomic can be removed (from https://app.circleci.com/pipelines/github/hashicorp/vault/15613/workflows/39275698-fbdc-49ea-89de-ea876aa43e4d/jobs/158313 )